pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -293,7 +319,8 @@ func (r *ReconcileNode) reconcileCrashRetention(namespace string, cephCluster ce
 		}
 	} else {
 		logger.Debugf("daysToRetain set to: %d", cephCluster.Spec.CrashCollector.DaysToRetain)
-		op, err := r.createOrUpdateCephCron(cephCluster, cephVersion)
+		useCronJobV1 := k8sVersion.AtLeast(version.MustParseSemantic(minVersionForCronV1))
```

Comment:
suggestion: move the `Atleast` method call at the beginning of this method, then `useCroneJobV1` can be used both in line 298 and 323

Reply:
done.